src/main/java/frc/robot/subsystems/Lights.java

```diff
@@ -0,0 +1,70 @@
+package frc.robot.subsystems;
+
+import edu.wpi.first.wpilibj.AddressableLED;
+import edu.wpi.first.wpilibj.AddressableLEDBuffer;
+import edu.wpi.first.wpilibj.util.Color8Bit;
+
+public class Lights {
+    // Docs -
+    // https://docs.wpilib.org/en/latest/docs/software/actuators/addressable-leds.html
+    private AddressableLED ledStrip;
+    private AddressableLEDBuffer ledBuffer;
+    private long delay;
+
+    private final Color8Bit RED = new Color8Bit(255, 0, 0);
+    private final Color8Bit BLUE = new Color8Bit(0, 0, 139);
+    private final Color8Bit LIME_GREEN = new Color8Bit(0, 255, 0);
+
+    public Lights(int pwmPort, int numLeds, long delay) {
+        ledStrip = new AddressableLED(pwmPort);
+        ledBuffer = new AddressableLEDBuffer(numLeds);
+        this.delay = delay;
+        ledStrip.setLength(ledBuffer.getLength());
+        ledStrip.setData(ledBuffer);
+        ledStrip.start();
+    }
+
+    public void allBlue() {
+        this.setColor(BLUE);
+    }
+
+    public void allLimeGreen() {
+        this.setColor(LIME_GREEN);
+    }
+
+    private void setColor(Color8Bit color) {
+        for (int x = 0; x < ledBuffer.getLength(); x++) {
+            ledBuffer.setLED(x, color);
+        }
+
+        ledStrip.setData(ledBuffer);
+    }
+
+    public void idleAnimation() {
+        for (int x = ledBuffer.getLength() - 2; x > 0; x--) {
```

Comment:
suggest making the 2 here be configurable.  Using the name DOT_SIZE as an example.

Then the following code would be:
for(int i = 0; i < DOT_SIZE; ++i)
{
    ledBuffer.setLED(ledBuffer.getLength() - x + i, RED;
    ledBuffer.setLED(x + i, RED);
}

And the same changes around line 56-59.

Reply:
Thanks for the suggestion, it is done